releasenotes/notes/change-rep-time-unit-48533d8fe474a035.yaml

```
@@ -0,0 +1,10 @@
+features:
+  - |
+    `assemble()` now takes `rep_time` in units of seconds, not microseconds.
+    `PulseBackendConfiguration._parse_pulse_args()` still takes `rep_times` in units of microseconds,
+    but they are now transformed to units of seconds before being passed to `assemble()`
+    At first pass, `PulseBackendConfiguration` will raise a warning to the user.
+deprecations:
```

This should probably be an upgrade note since it's not something that users have to update now:

```suggestion
upgrade:
```